server/src/main/resources/openapi/schemas/solrOutlet.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "Solr outlet configuration",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Solr outlet identifier"
+    },
+    "solrUrl" : {
+      "type" : "string",
+      "description" : "Solr server URL"
+    },
+    "transformer": {
+      "type": "string",
+      "description" : "OAI server response transformer invocation: 'moduleId<::function>' where <::function> is optional"
+    },
+    "matchkeyId" : {
```

Comment:
@adamdickmeiss I wonder if we should invert the relationship. E.g if it's more convenient to define a SOLR outlet once and then refer to it in the "pool" configuration. But I am not sure, so let's discuss.

Reply:
I'd not do that. We could have multiple Solr outlets for one pool. It also makes it more difficult to make Solr Outlet a separate module (if we want that).